packages/wallet/src/KeyManagement/types.ts

```diff
@@ -49,16 +59,40 @@ export enum CommunicationType {
 
 export type BIP32Path = Array<number>;
 
-export interface AccountAddressDerivationPath {
-  type: AddressType;
-  index: number;
+export enum AccountDerivationPathDefaults {
+  /**
+   * Default: 1852
+   */
+  Purpose = CardanoKeyConst.PURPOSE,
+  /**
+   * Default: 1815
+   */
+  CoinType = CardanoKeyConst.COIN_TYPE,
+  AccountIndex = 0
 }
 
-export interface GroupedAddress {
+export interface AccountDerivationPath {
+  accountIndex: number;
+  purpose: number;
+  coinType: number;
+}
+
+export type AccountKeyDerivationPath = {
+  role: KeyRole;
+  index: number;
+};
+
+export type AccountAddressDerivationPath = {
   type: AddressType;
   index: number;
+};
+
+export interface GroupedAddress {
   networkId: Cardano.NetworkId;
-  accountIndex: number;
+  derivationPath: AccountDerivationPath & {
```

Comment:
This will be a breaking change with loading from pouchdb stores. Let's rename the store to `'addresses_v2'` [here](https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/wallet/src/persistence/pouchdbStores/pouchdbWalletStores.ts#L50)

Reply:
This is now a non-issue right?